pages/linux/trayer.md

```diff
@@ -0,0 +1,28 @@
+# trayer
+
+> A lightweight GTK-2 based systray for UNIX desktop.
```

Comment:
```suggestion
> A lightweight GTK-2 based systray.
```
IMO we don't need to specify `for UNIX desktop` as we are moving away from this naming in other platform pages.